prowlarr/umbrel-app.yml

```
@@ -21,8 +21,11 @@ path: ""
 defaultUsername: ""
 defaultPassword: ""
 releaseNotes: >-
-  On installation of Sonarr, Lidarr or Radarr, Prowlarr will now automatically setup the app's connection within its settings
+  On installation of Sonarr, Lidarr or Radarr, Prowlarr will automatically setup the app's connection within its settings
+
+
+  A full list of new features, improvements, and bug fixes: https://github.com/linuxserver/docker-prowlarr/releases
```

We can remove the set up instructions from the previous release notes.

```suggestion
  Full release notes can be found here: https://github.com/linuxserver/docker-prowlarr/releases
```